governance/SDK.md

```diff
@@ -0,0 +1,172 @@
+
+It may happen that, if a project is not following the criteria 1, 2, 4, 5, 6, 7,
+8, the community might decide to handover the project to a new maintainer/group
+of maintainers. The community can perform the handover to a new maintainer
+**iff** a voting process to add such maintainer was **already performed** with
+100% of entitled people abstained and:
+
+- Or the new maintainer(s) already meet the requirements enlisted in
+  [New maintainers section](#new-maintainers),
+- Or, because of the inactivity of the project due to criteria 7, the new
+  maintainer(s) performed the required PRs/Reviews/Code changes to meet the
+  [New maintainers criteria](#new-maintainers) but he/they wasn't/weren't able
+  to merge them.
+
+The new maintainer must show the good will to keep the project healthy and
+community could ask for an eventual plan to improve the quality of the project.
+
+As soon as the new maintainer is identified, the community can proceed to vote
+to handover the project using the
+[Asynchronous voting process](#asynchronous-voting-process). If more than one
+new maintainer wants to handover the project, separate voting process must be
+done for all of them, and both of them must follow the same voting criteria (and
+not the ones from [New maintainers section](#new-maintainers)).
+
+The voting criteria are:
+
+- 1 week to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+### Archive a project
+
+It may happen that, if a project is not following the criteria 1, 2, 4, 5, 6, 7,
+8, the community might decide to archive the project. Prior to archiving the
+project, the community should consider first to search a new maintainer to
+[handover the project](#handover-to-a-new-maintainergroup-of-maintainers).
+
+If no new maintainer is found, the community can proceed to archive the project
+using the [Asynchronous voting process](#asynchronous-voting-process).
+
+The voting criteria are:
+
+- 2 weeks to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+## Asynchronous voting process
+
+In order to ensure the democratic management of debates in the community, we
+define an asynchronous voting process using GitHub issues.
+
+The voting is defined by:
+
+- The proposed community change to vote
+- The set of people/groups entitled to vote
+- A voting period
+- The percentage of agreements required to accept the change
+- A member of the community entitled to manage the vote, called voting manager.
+  The voting manager MUST also be part of people entitled to vote.
+
+To start a vote, the voting manager opens an issue in the
+[CloudEvents Spec](https://github.com/cloudevents/spec/issues) with:
+
+- The binary question which explains the change
+- The tags to the entitled people/groups to vote (depending on the question)
+- The expiration date of the vote
+- The percentage of agreements to accept the change
```

Comment:
```suggestion
- A question to be voted on
- Tags of the entitled people/groups to vote (depending on the question)
- The expiration date of the vote
- The percentage of votes required to accept the change
```

Does this voting process only take into account binary questions? Here is a suggestion that allows for a more flexible voting mechanism.

Reply:
> Does this voting process only take into account binary questions?

I would prefer to keep the binary questions, they're easier to manage and don't leave room to "confusion" on the vote result. Also at least in this document I don't see any use case for non-binary questions, we could always evolve such process later when we need it